### PR TITLE
fixing bug: read utf8 format error : orchestration/experimental/interactive/interactive_context.py 

### PR DESCRIPTION
**Pull request by m2kar**
## Error Occur
When I run 
`https://github.com/tensorflow/tfx/blob/master/docs/tutorials/tfx/components_keras.ipynb`

There raise a `UnicodeDecodeError`.

### Error Trace
image

### Bug File Position

`tfx/orchestration/experimental/interactive/interactive_context.py``
Line 191-192
### Bug Fix Way
add `utf-8` support to `open()` method

### CODE REVIEW

**Comment by m2kar** [comment]
@googlebot I signed it!

**Comment by m2kar** [comment]
meet pylint errors, pull requests in #1912. 

```
[NOTE] This linter is currently EXPERIMENTAL.=======================================
Please contact reviewers for existing lint errors or false negative errors.
Feel free to send PRs for pylintrc in the root directory of the repository if needed.
====================================================================================
************* Module tfx.orchestration.experimental.interactive.interactive_context
tfx/orchestration/experimental/interactive/interactive_context.py:61:2: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
tfx/orchestration/experimental/interactive/interactive_context.py:229:4: C0415: Import outside toplevel (IPython.core.display.display) (import-outside-toplevel)
tfx/orchestration/experimental/interactive/interactive_context.py:230:4: C0415: Import outside toplevel (IPython.core.display.HTML) (import-outside-toplevel)
##[error]Process completed with exit code 24.
```

**Comment by chongkong** [comment]
Hi m2kar, so sorry that I come to the PR so lately. Our oncall system was not functioning at the moment and we missed your contribution.

File: tfx/orchestration/experimental/interactive/interactive_context.py [hunk 1]
```diff
@@ -188,8 +188,8 @@ def export_to_pipeline(self, notebook_filepath: Text, export_filepath: Text,
     absl.logging.info('Exporting contents of %s to %s with %s runner.',
                       notebook_filepath, export_filepath, runner_type)
 
-    with open(notebook_filepath) as notebook_f,\
-        open(export_filepath, 'w') as export_f:
+    with open(notebook_filepath, encoding="utf-8") as notebook_f,\
```
Comment (chongkong) [inline tfx/orchestration/experimental/interactive/interactive_context.py:191]: nit: Please use single quote

File: tfx/orchestration/experimental/interactive/interactive_context.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def run_if_ipython(*args, **kwargs):
       absl.logging.warning(
           'Method "%s" is a no-op when invoked outside of IPython.',
           fn.__name__)
-
+    return None
```
Comment (chongkong) [inline tfx/orchestration/experimental/interactive/interactive_context.py:71]: nit: Maybe put this inside else block?

**Comment by m2kar** [comment]
@chongkong I've fixed. Please review it . Thank you.

**Comment by m2kar** [comment]
@chongkong could this PR merge to master ?

**Comment by m2kar** [comment]
@chongkong could this PR merge to master ?

**Review by zhitaoli — COMMENTED** [review]
This seems still a real issue.

@charlesccychen maybe you can take a look?

**Comment by charlesccychen** [comment]
We are planning to deprecate the export functionality and remove it from the tutorials for now, so this issue will be fixed.

**Pull request closed without merging**